// 関数宣言
console.log(`3 + 5 = ${sum(3, 5)}`);

function sum(val1, val2) {
  return val1 + val2;
}
console.log(`10 + 7 = ${sum(10, 7)}`);

// 関数式
// console.log(`3 + 5 = ${sum(3, 5)}`); // エラーが発生！

// // エラーが発生したため、これ以降は実行されない
// const sum = function (val1, val2) {
//   return val1 + val2;
// };
// console.log(`10 + 7 = ${sum(10, 7)}`);
